### PR TITLE
Update nuxt and styleguide

### PR DESCRIPTION
**Pull request by delrueba**

### CODE REVIEW

File: assets/sass/main.scss [hunk 1]
```diff
@@ -1,142 +1,6 @@
...
-// inject:layouts
-@import "../styleguide/sass/61-layouts/general/_layouts.scss";
-@import "../styleguide/sass/61-layouts/general/detail-layout/_detail-layout.scss";
-@import "../styleguide/sass/61-layouts/general/overview-layout/_overview-layout.scss";
-@import "../styleguide/sass/61-layouts/general/sidebar-layout/_sidebar-layout.scss";
-// endinject
-
-@import "molecules/teaser-search";
-@import "organisms/header";
+@import "~gent_styleguide/build/styleguide/sass/main_cli";
```
Comment (hstandaert) [inline assets/sass/main.scss]: 🥇

File: components/molecules/gent-info.vue [hunk 2]
```diff
@@ -1,19 +1,23 @@
 <template>
   <div class="gentinfo-block">
     <div class="gentinfo-block-content">
-      <h2>Gentinfo</h2>
-      <p class="">Bereikbaar van maandag t.e.m. zaterdag van 8:00 tot 19:00 uur.</p>
+      <h2>
+        <img src="~assets/styleguide/img/svg/gentinfo-logo.svg" alt="Gentinfo" class="gentinfo-logo">
+      </h2>
+      <p class="">
```
Comment (hstandaert) [inline components/molecules/gent-info.vue]:
```suggestion
      <p>
```

File: components/molecules/partnership.vue [hunk 3]
```diff
@@ -1,6 +1,8 @@
 <template>
   <div class="db-block-partnership">
-    <p class="">In samenwerking met Digipolis</p>
-    <i aria-hidden="true" class="icon-digipolis"/>
+    <p class="">
```
Comment (hstandaert) [inline components/molecules/partnership.vue]:
```suggestion
    <p>
```

File: components/molecules/social-media.vue [hunk 4]
```diff
@@ -3,19 +3,37 @@
...
-        <li><a href="https://www.linkedin.com/company/stad-gent"><i aria-hidden="true"
-                                                                    class="icon-linkedin"/><span
-                                                                      class="visually-hidden">LinkedIn</span></a></li>
-        <li><a href="https://www.youtube.com/channel/UCmJLf8SFG0D0GTd6LSSTQLw"><i aria-hidden="true"
-                                                                                  class="icon-youtube"/><span
-                                                                                    class="visually-hidden">Youtube</span></a></li>
-        <li><a href="https://www.instagram.com/stadgent/"><i aria-hidden="true" class="icon-instagram"/><span
-          class="visually-hidden">Instagram</span></a></li>
+        <li>
+          <a href="https://www.facebook.com/stadsbestuurgent"><i aria-hidden="true"
```
Comment (hstandaert) [inline components/molecules/social-media.vue]: Formatting should be improved for this file. This isn't exactly readable 😅

```html
<li>
	<a href="https://www.facebook.com/stadsbestuurgent">
		<i aria-hidden="true" class="icon-facebook"/>
		<span class="visually-hidden">Facebook</span>
	</a>
</li>
```

File: layouts/error.vue [hunk 5]
```diff
@@ -5,37 +5,48 @@
       <h2>Hoe kunnen we u verder helpen?</h2>
       <div class="help-block-section">
         <h3>1. Verder navigeren vanaf hier?</h3>
-        <p class="">Dat kan op verschillende manieren:</p>
+        <p class="">
```
Comment (hstandaert) [inline layouts/error.vue]:
```suggestion
        <p>
```

File: layouts/error.vue [hunk 6]
```diff
@@ -5,37 +5,48 @@
...
   <article v-else-if="error.statusCode === 403" class="detail-layout">
     <h1>Geen toegang tot deze pagina</h1>
     <div class="search-block box-left">
       <div class="search-block-content">
         <h2>Vanaf hier kan u...</h2>
         <ul class="icon-list">
-          <li><i class="icon-home" aria-hidden="true"/><span>Terugkeren naar de <nuxt-link :to="'/'" >homepage</nuxt-link></span></li>
-          <li><i class="icon-fancyback" aria-hidden="true"/><span>De terugknop van uw browser gebruiken</span></li>
+          <li>
+            <i class="icon-home" aria-hidden="true" /><span>Terugkeren naar de
```
Comment (hstandaert) [inline layouts/error.vue]: Improve formatting:

```html
<li>
	<i class="icon-home" aria-hidden="true" />
	<span>Terugkeren naar de <nuxt-link :to="'/'">homepage</nuxt-link></span>
</li>
```

File: layouts/error.vue [hunk 7]
```diff
@@ -62,6 +78,9 @@ export default {
       required: true
     }
   },
-  layout: "default"
+  layout: 'default',
+  mounted () {
+    console.warn(JSON.parse(JSON.stringify(this.error)))
```
Comment (hstandaert) [inline layouts/error.vue]: Is this for testing purposes? 🤔

File: package.json [hunk 8]
```diff
@@ -17,28 +17,41 @@
     "lint"
   ],
   "dependencies": {
-    "@nuxtjs/google-analytics": "^2.0.2",
+    "@nuxtjs/axios": "^5.3.6",
+    "@nuxtjs/google-analytics": "^2.2.0",
     "axios": "^0.18.0",
```
Comment (hstandaert) [inline package.json]: Should `axios` still be a direct dependency? It's a dependency of [@nuxtjs/axios](https://www.npmjs.com/package/@nuxtjs/axios), so unless explicitly used in the project somewhere, this dependency should be removed.

```suggestion
```

File: pages/index.vue [hunk 9]
```diff
@@ -1,77 +1,82 @@
 <template>
   <section class="content-container">
     <h1 class="overview-title">
-      <span class="overview-title-colon">Lijst verwerkingen van persoonsgegevens<span class="colon" aria-hidden="true"/></span>
+      <span class="overview-title-colon">
```
Comment (hstandaert) [inline pages/index.vue]: Improve formatting

File: pages/medewerkers/index.vue [hunk 10]
```diff
@@ -1,81 +1,86 @@
 <template>
   <section class="content-container">
     <h1 class="overview-title">
-      <span class="overview-title-colon">Lijst verwerkingen van persoonsgegevens voor medewerkers<span class="colon" aria-hidden="true"/></span>
+      <span class="overview-title-colon">
+        Lijst verwerkingen van persoonsgegevens voor medewerkers<span class="colon" aria-hidden="true" /></span>
```
Comment (hstandaert) [inline pages/medewerkers/index.vue]: Improve formatting

File: pages/medewerkers/index.vue [hunk 11]
```diff
@@ -1,81 +1,86 @@
...
     </h1>
     <aside class="box-top">
       <div class="inner-box">
-        <introductietekst/>
-        <div class="accolade "/>
+        <introductietekst />
+        <div class="accolade " />
```
Comment (hstandaert) [inline pages/medewerkers/index.vue]:
```suggestion
        <div class="accolade" />
```

File: package.json (hunk 8 above)
Comment (delrueba) [reply package.json]: ... seems to work just fine if I remove it.

File: components/molecules/social-media.vue (hunk 4 above)
Comment (delrueba) [reply components/molecules/social-media.vue]: Yes... the linting rules have changed.
Guess lintfix wasn't the prettiest solution.

**Pull request merged**